### PR TITLE
use gpu metrics

### PR DESCRIPTION
**Pull request by tgerdesnv**
Enable MA to use PA metrics

### CODE REVIEW

File: model_analyzer/perf_analyzer/perf_analyzer.py [hunk 1]
```diff
@@ -187,14 +188,23 @@ def get_records(self):
         """
```
Comment (nv-braf) [inline model_analyzer/perf_analyzer/perf_analyzer.py:188]: Should this be renamed to get_perf_records?

File: model_analyzer/perf_analyzer/perf_config.py [hunk 2]
```diff
@@ -150,6 +152,12 @@ def update_config_from_profile_config(self, model_name, profile_config):
             'verbose-csv': '--verbose-csv'
         }
 
+        params.update({
+            'collect-metrics': 'True',
+            'metrics-url': profile_config.triton_metrics_url,
+            'metrics-interval': profile_config.monitoring_interval * 1000
```
Comment (nv-braf) [inline model_analyzer/perf_analyzer/perf_config.py:158]: Get rid of magic number

Comment (tgerdesnv) [reply model_analyzer/perf_analyzer/perf_config.py]: Done

File: model_analyzer/perf_analyzer/perf_analyzer.py (hunk 1 above)
Comment (tgerdesnv) [reply model_analyzer/perf_analyzer/perf_analyzer.py:188]: Done

**Review by nv-braf — DISMISSED** [review]

**Review by nv-braf — APPROVED** [review]

**Pull request merged**